tsl/test/shared/expected/ordered_append_join.out

```diff
@@ -3671,14 +3670,14 @@ FROM :TEST_TABLE o1
     AND o1.time = o2.time
   ORDER BY o1.time
   LIMIT 100;
-                                                                                            QUERY PLAN                                                                                            
---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
+                                                                                               QUERY PLAN                                                                                               
+--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
  Limit (actual rows=100 loops=1)
```

Comment:
Hmm this plan looks worse now with this change

Reply:
Hmm Limit -> Sort is usually not favorable plan as the Sort prevents further pushdown of the limit

Reply:
I assume it looked at both paths and found there wasn't enough benefit to pushing down the limit.  We actually run this same query on four different tables and it still uses the merge join in two of the cases (in the other case, both before and after this change it uses a Limit->Sort).